mvp-spec/operator-api.md

```diff
@@ -16,13 +16,13 @@ These endpoints need to support both the "3PC" and "no 3PC" contexts.
-| --------------------------------------- | ----------------------- | ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- | ------------------------------- |
-| Read data                               | -                       | List of IDs (if any).<br>List of preferences (if any) | If ID 🍪 exist, return it.<br>If preferences 🍪 exist, return them                                                                                                                        | -                                                                                                                                                                                                                                                                                       | GET /v1/json/read           | GET /v1/redirect/read           |
-| Read data if exists, or return a new Id | -                       | List of IDs.<br>List of preferences (if any)          | If ID 🍪 exist, return it.<br><br>Otherwise: create a new ID, sign it and return it. In this case, the **new ID is not stored in a cookie yet**..<br>If preferences 🍪 exist, return them | Why don't we save the newly generated ID?<br>Because haven't yet received consent. The new ID will be saved as cookie, along with preferences, when it is provided via the "Write data" endpoint ⬇️                                                                                     | GET /v1/json/readOrGetNewId | GET /v1/redirect/readOrGetNewId |
-| Write data.<br>and return written data  | main ID.<br>preferences | List of IDs.<br>List of preferences                   | Read provided ID.<br>Verify provided signatures.<br>Write ID 🍪.<br>Write preferences 🍪                                                                                                  | ID is mandatory input for the "first visit" use case so is considered always mandatory, for consistency.<br>Data is written because it will be used for confirmation.                                                                                                                   | POST /v1/json/write         | GET /v1/redirect/write          |
```

Comment:
We mention a mandatory input, here. However, it doesn't appear in the detail of the API.
Do we want to be precise here ? What do you think about adding a payload for the POST and a URL param (or a path) for the GET? Or maybe in URL for both for consistency?